themes/pycon-ca-2018/templates/base.tmpl

```diff
@@ -20,12 +20,6 @@
     <!-- Navigation bar -->
     <nav class="navbar sticky-top navbar-expand-lg navbar-dark bg-dark-cyan">
 
-        <!-- Brand + icon -->
-        <a class="navbar-brand" href="{{ abs_link(_link("root", None, lang)) }}">
```

Comment:
Removing this means we don't have a home link at the top nav. We could add it back inside conf.py.